opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraphBar.jsx

```diff
@@ -929,6 +914,40 @@ class GroupingKnowledgeGraphBar extends Component {
                         enableReferences={enableReferences}
                       />
                     )}
+                    {handleToggleQueryMode && (
+                    <ToggleButtonGroup
+                      size="small"
+                      value={currentQueryMode}
+                      exclusive
+                      onChange={handleToggleQueryMode}
+                      style={{
+                        padding: '4px 8px',
```

Comment:
The ToggleButtonGroup is not "perfectly" centered:
![image](https://github.com/user-attachments/assets/cafaad75-00f5-45c0-b1c9-72af89609b65)

```suggestion
                        padding: '6px 8px',
```
After: 
![image](https://github.com/user-attachments/assets/e4f7210a-c7b9-436d-ad4e-01b36814a185)